### ISSUE TITLE
Regression: @throws and @example missing from output

### ISSUE THREAD

**Issue by 75lb** [jsdoc3/jsdoc#1414]
This may be related to https://github.com/jsdoc3/jsdoc/issues/1129.

In jsdoc v3.5.3, the documentation for `@throws` and `@example` below is not included in the generated html output. It worked in v3.4.

```js
/**
 * A class.
 */
class Something {
  /**
   * Might throw something.
   * @throws something
   * @example
   * const one = 1
   */
  constructor () {}
}
```

This is what the explain output looks like: 

```json
[
    {
        "comment": "/**\n * A class.\n */",
        "meta": {
            "range": [
                20,
                155
            ],
            "filename": "throws-constructor.js",
            "lineno": 4,
            "columnno": 0,
            "path": "/Users/lloydb/Documents/jsdoc2md/testbed/buggy",
            "code": {
                "id": "astnode100000002",
                "name": "Something",
                "type": "ClassDeclaration",
                "paramnames": []
            }
        },
        "classdesc": "A class.",
        "name": "Something",
        "longname": "Something",
        "kind": "class",
        "scope": "global",
        "description": "Might throw something.",
        "params": []
    },
    {
        "comment": "/**\n   * Might throw something.\n   * @throws something\n   * @example\n   * const one = 1\n   */",
        "meta": {
            "range": [
                136,
                153
            ],
            "filename": "throws-constructor.js",
            "lineno": 11,
            "columnno": 2,
            "path": "/Users/lloydb/Documents/jsdoc2md/testbed/buggy",
            "code": {
                "id": "astnode100000005",
                "name": "Something",
                "type": "MethodDefinition",
                "paramnames": []
            },
            "vars": {
                "": null
            }
        },
        "description": "Might throw something.",
        "exceptions": [
            {
                "description": "something"
            }
        ],
        "examples": [
            "const one = 1"
        ],
        "name": "Something",
        "longname": "Something",
        "kind": "class",
        "scope": "global",
        "params": [],
        "undocumented": true
    },
    {
        "kind": "package",
        "longname": "package:undefined",
        "files": [
            "/Users/lloydb/Documents/jsdoc2md/testbed/buggy/throws-constructor.js"
        ]
    }
]
```

**Comment by hegemonic** [comment]
Hmmm. I don't think this is a regression. I just tested your example in JSDoc 3.4.3, and as expected, the `@throws` and `@example` info was missing from the HTML output. JSDoc 3.4.3 also gives me the same `--explain` output that you got. Can you double-check your JSDoc versions and try again?

(I also confirmed that on GitHub master, the `@throws` and `@example` info _is_ present in the HTML output. That's because of the fix for #1129, as you surmised, which will be released in JSDoc 3.6.0.)

**Comment by 75lb** [comment]
hi, sorry - i haven't had chance double-check yet but closing anyway since you've fixed it on master.. thanks for recent improvements 👍
